dotCMS/src/main/webapp/WEB-INF/messages/Language.properties

```diff
@@ -1193,6 +1193,7 @@ dot.common.message.locking=Locking...
 dot.common.message.pageurl.copy.clipboard=Copy Page URL
 dot.common.message.saved=All changes Saved
 dot.common.message.saving=Saving...
+dot.common.message.success=Succes
```

Comment:
Here is a typo. `Success`* Also there is `Success=Success!` not sure if we want it without the exclamation mark